extra/release_tool.py

```diff
@@ -172,6 +173,7 @@ def get_components_of_type(
             is_release_component = Component.COMPONENT_MAPS[type][comp][
                 "release_component"
             ]
+            non_core_component = Component.COMPONENT_MAPS.get(type,{}).get(comp, {}).get("non_core_component", False)
```

Comment:
Nitpick: for consistency with the others I suggest renaming to `is_non_core_component`

Reply:
I'm probably misunderstanding, but the other's are named:
* `independent_component`
* `release_component`
Where does the `is` come in?

Reply:
I meant in the code, the Python variable. Two lines above you have `is_release_component = ...`.